projects/teleport/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 Google Inc.
+# Copyright 2022 Google LLC
```

Comment:
No need to update the year.

Reply:
This came from us started from scratch as part of our native Go fuzzing upgrade. I've reverted the year change (though kept the legal entity update, as "Google Inc." is gone as of 2017).